backend/schemas.py

```diff
@@ -202,6 +208,12 @@ class CreatePartnerMemberSchema(BaseModel):
     is_active: Optional[bool] = True
 
 
+class CreateOfficerSchema(_BaseCreateOfficerSchema, _OfficerMixin):
+    known_employers: Optional[List[CreateEmploymentSchema]]
+    accusations: Optional[List[CreateAccusationSchema]]
+    state_ids: Optional[List[CreateStateIDSchema]]
+
+
```

Comment:
Why do we have two identical schemas here except for optionality (CreateOfficerSchema and OfficerSchema)?

Reply:
They aren't entirely identical. The create version (`schema_create`) just excludes the ids. There's probably a cleaner way of doing it, but I haven't looked into it. Much of this code was inherited from an earlier cohort who have mostly moved on to other projects or left CfB.